src/main/resources/common/sitemap-entry.jsp

```diff
@@ -39,9 +39,19 @@
     <c:if test="${locales.size() > 1}"><%-- no need for alt if there's only one locale --%>
         <c:forEach var="locale" items="${locales}">
             <c:set var="lang" value="${locale.toString()}"/>
-            
+
+            <c:set var="isValidLanguage" value="true"/>
+            <c:if test="${urlNode.hasProperty('j:invalidLanguages')}"><%-- the alternate languages should not list invalid language --%>
+                <jcr:nodeProperty node="${urlNode}" name="j:invalidLanguages" var="invalidLanguages"/>
+                <c:forEach items="${invalidLanguages}" var="invalidLanguage">
+                    <c:if test="${invalidLanguage eq lang}">
+                        <c:set var="isValidLanguage" value="false"/>
+                    </c:if>
+                </c:forEach>
```

Comment:
Can be replace by: 

```
<c:if test="${functions:contains(invalidLanguages, lang)}">
      <c:set var="isValidLanguage" value="false"/>
</c:if>
```

using taglib:
`<%@ taglib prefix="functions" uri="http://www.jahia.org/tags/functions" %>`